src/components/vr-scatter-plot/vr-scatter-plot.component.ts

```diff
@@ -9,16 +9,21 @@ import { LineChartMeta } from '../../datasets/metas/line-chart.meta';
 import { BehaviorSubject, Subject } from 'rxjs';
 import { DAY } from '../../utils/timeUnits';
 import { map, takeUntil } from 'rxjs/operators';
+import { DATA_PREFERENCE } from '../../i18n';
+import { datasets } from '../../datasets';
 
 @Component({
   selector: 'app-vr-scatter-plot',
-  templateUrl: './vr-scatter-plot.component.html'
+  templateUrl: './vr-scatter-plot.component.html',
+  styleUrls: ['./vr-scatter-plot.component.scss'],
 })
 export class VRScatterPlotComponent<T extends Meta>implements OnInit, OnChanges, OnDestroy{
   @Input() endDate = new Date();
   @Input() startDate = new Date(this.endDate.getTime() - 30 * DAY);
-  @Input() meta: Meta;
+  @Input() meta: Meta | Meta[];
```

Comment:
Could you make these have a more descriptive name.

Reply:
https://github.com/googleinterns/guide-doge/pull/57#discussion_r456192492